CMakeLists.txt

```diff
@@ -152,15 +179,12 @@ set(FOLLY_LINK_LIBRARIES
 
 target_include_directories(folly_base
   PUBLIC
-    ${DOUBLE_CONVERSION_INCLUDE_DIR}
-    ${LIBGFLAGS_INCLUDE_DIR}
-    ${LIBGLOG_INCLUDE_DIR}
-    ${LIBEVENT_INCLUDE_DIR}
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
```

Comment:
This line is still needed, for folly-config.h.

Reply:
Ok

Reply:
I returned it